controllers/user/user.go

```diff
@@ -0,0 +1,117 @@
+package user
+
+import (
+	userModel "github.com/divisi-developer-poros/poros-web-backend/models/user"
+	"github.com/divisi-developer-poros/poros-web-backend/util/response"
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+
+	"net/http"
+	"strconv"
+)
+
+func GetAll(c *gin.Context) {
+	var users []userModel.User
+
+	if err := userModel.GetAll(&users); err != nil {
+		response.Res(c, "Content-Type", "application/json", "error", "failed when fetching users", http.StatusBadRequest, err)
+		return
+	} else {
+		response.Res(c, "Content-Type", "application/json", "success", "null", http.StatusOK, users)
+		return
+	}
+
+}
+
+func Get(c *gin.Context) {
+	id := c.Params.ByName("id")
+
+	if numId, error := strconv.Atoi(id); error != nil {
+		response.Res(c, "Content-Type", "application/json", "error", "ID not valid", http.StatusBadRequest, nil)
+		return
+	} else {
+		var user userModel.User
+
+		if err := userModel.Get(&user, numId); err != nil {
+			response.Res(c, "Content-Type", "application/json", "error", "user not found", http.StatusNotFound, err)
+			return
+		} else {
+			response.Res(c, "Content-Type", "application/json", "success", "null", http.StatusOK, user)
+			return
+		}
+	}
+}
+
+func Create(c *gin.Context) {
+	var user userModel.User
+	c.BindJSON(&user)
```

Comment:
wow, i did the same binding method in my testing handlers. I thinks i would create an options for binding method because behaviours of `BindJSON()` will always in partner with `AbortWithError()`. Thanks for the insight! 

[see more](https://github.com/gin-gonic/gin#model-binding-and-validation) for BindJSON() in gin-gonic